### ISSUE TITLE
Radius Logs in Wrong Timezone

### ISSUE THREAD

**Issue by simonmorley** [cucumber-tony/cucumber-frontend#191]
We fixed the timezone for the boxes but didn't adjust for the radius logs.

<img width="338" alt="screen shot 2016-07-14 at 20 19 29" src="https://cloud.githubusercontent.com/assets/309336/16852525/607414f8-4a00-11e6-9cb3-aaff7830d31a.png">

**Comment by simonmorley** [comment]
Apparently clients are off too

**Comment by simonmorley** [comment]
Clients now look good.

The radius logs are weird, they're still off despite setting the TZ

**Comment by simonmorley** [comment]
This was caused by the radius server writing the time into MySQL in the local time of the server and not UTC.

We've implemented a fix on the radius servers to insert in the correct zone again.

Currently, the localtime for the user (based on browser) is applied to the sessions. However, if there's two locations showing in two timezones, one set of results (or both in reality) will be wrong.

I'd prefer to apply a filter to the sessions so they showed the results from a single location only. Then we can apply the filter to the results properly.
